package.json

```diff
@@ -1253,6 +1253,19 @@
           "default": [],
           "scope": "resource"
         },
+        "cmake.ctestDefaultArgs": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "%cmake-tools.configuration.cmake.ctestDefaultArgs.description%",
+          "default": [
+            "-T",
+            "test",
+            "--output-on-failure"
+          ],
+          "scope": "resource"
```

Comment:
Can you set the scope to `machine`?  I think this will help the setting make more sense to people.  This setting is meant to apply to all instances of vscode on the machine and should not be overridable by the workspace or folder.

Reply:
Done!